codec-http/src/main/java/io/netty/handler/codec/http/HttpObjectDecoder.java

```diff
@@ -905,13 +905,16 @@ private static int getChunkSize(byte[] hex, int start, int length) {
                         // empty case
                         throw new NumberFormatException();
                     }
-                    return result;
+                    break;
```

Comment:
doesn't this change the return value ?

Reply:
i dont think so? it should leave the loop and also return result there, just with the additional check.

i guess this isn't needed anymore since i moved the check inside the loop.